delivery_carrier_general_manifest/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery Carrier General Manifest",
+    "summary": "Manifest files for all carriers",
+    "version": "17.0.1.0.0",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "maintainers": ["tisho99"],
```

Comment:
are u sure about that?

Reply:
No, i'll delete it if you want

Reply:
It's no problem for me. Do you know what it means to be there?

Reply:
I know something, like I can merge PRs from this module. Is there any documentation of the maintainer duties?

Reply:
As far as I know, there's no documentation for that part. In short, it's this: you agree to be the maintainer of this module, meaning you'll be automatically pinged if someone opens a PR for this module.